src/app/edit-recipe/edit-recipe.component.ts

```diff
@@ -135,17 +135,25 @@ export class EditRecipeComponent implements OnInit {
       measures.push(e.measure);
     });
     const recipeToUpdate = {
-      idMeal: this.recipeId.toString(),
+      recipeId: this.recipeId.toString(),
       strMeal: this.name,
       strTime: this.time + 'min',
-      strDescription: this.description,
+      strDescription:  this.description,
       strInstructions: this.instruction,
-      strMealThumb: this.image,
+      strMealThumb:  this.image,
       strIngredients: ingredients,
       strMeasures: measures,
     };
     this.firebaseService.updateRecipe(this.recipeId, recipeToUpdate)
-      .then(() => this.router.navigate(['/recipes']));
+      .then((ref) => {
+        console.log(ref)
+        this.router.navigate(['/recipes'])});
+  }
+  onDeleteRecipe(){
+    this.firebaseService.deleteRecipe(this.recipeId)
+      .then((ref)=>{
```

Comment:
Remember to use autoformatter! Should be `.then((ref) => {` or can be even shortened to `.then(ref => {`